yesod-core/Yesod/Core/Dispatch.hs

```diff
@@ -17,13 +17,16 @@ module Yesod.Core.Dispatch
     , mkYesodSubData
     , mkYesodDispatch
     , mkYesodSubDispatch
+      -- *** Helpers
```

Comment:
If we're going to add new exports, they need to have Haddocks and `@since` comments. Can you add those, and do a minor version bump in the .cabal file?